### ISSUE TITLE
Maven artifacts for releases after 3.3.7

### ISSUE THREAD

**Issue by hbs** [processing/processing#6006]
## Description

The last release with published artifacts seems to be 3.3.7, would it be possible to publish more recent artifacts?

For all uses of the processing core library in JDK >= 9, the fix from #5626 is needed, and this today requires a recompiling of the core artifact and a distribution on an alternate maven repo.

Is there a particular reason why 3.3.7 is the last one available?

## Steps to Reproduce

1.
2.
3.

## Your Environment

 

* Processing version:
* Operating System and OS version:
* Other information:

**Comment by pi-r-p** [comment]
Hello dear maintainers,

If the missing packages on maven are linked to the broken links from oracle in the publication toolchain, I can push a PR to `/build/jre/src/Downloader.java`, to make it point to https://github.com/frekele/oracle-java/releases. This will fix the "ant run". Should I ?

**Comment by monkstone** [comment]
@hbs There is no official support for the maven releases, these were created by @poqudrof for use in his projects such as https://github.com/poqudrof/PapARt and kindly shared for everyone to use. It would be nice to have such artefacts officially supported.

**Comment by atzeus** [comment]
I would also like newer releases on Maven central. Any chance of this happening? Thanks!

**Comment by jeremydouglass** [comment]
Not sure if this is relevant, but on the forum, @neilcsmith-net recently mentioned
https://discourse.processing.org/t/processing-in-style-with-java-11/13776/32?u=jeremydouglass

> have split Processing Core into multiple explicit modules. These will be on Maven Central very soon - working on that now.

**Comment by hbs** [comment]
I am not convinced that shifting to modules should be a prereq to supporting JDK9+. The core component of Processing (org.processing:core as it is known on Maven Central) works fine in JDK9+ after #5626 was merged, but it also works on JDK8, whereas going the module route would make it non JDK8 compatible. So I am in favor of releasing a 3.5.4 on Maven Central, ideally under the org.processing group.

Maybe the real question is who has the credentials for this group?

**Comment by neilcsmith-net** [comment]
@hbs this work is for our own purposes, not for release under this namespace. The reason for explicit modules would be to properly support LWJGL when on the module path. I'm personally in favour of JPMS for Processing 4 though, and that will require Java 11+ anyway.

**Comment by jeremydouglass** [comment]
> Maybe the real question is who has the credentials for this group?

Last time I saw it discussed ben said the maven releases were always "unofficial".

From https://github.com/processing/processing/issues/5666#issuecomment-426453307 :

> It's not done officially. I wish people wouldn't do it, because now I'm spending time answering questions about a release that's not supported. Which is… not supported because I don't have the time.
>

**Comment by poqudrof** [comment]
Hello, 
I did write the maven distribution a few years ago. Unfortenately I create less and less Processing sketches and application and I could not maintain it in the past years. 
So 3.3.7 is still my reference version for my projects I did not update since, I know it is kind of a lame reason to be stuck !
I think I still have the repositiories / code editor on my laptop I could have a look.
However, the deployment right is not that hard to obtain and my maven configuration was not the most complex one. So if anyone want to push new versions you are welcome !

Cheers, 
Jeremy.
